modin/pandas/test/utils.py

```diff
@@ -977,7 +978,7 @@ def applyier(module, *args, **kwargs):
         return result
 
     def call_eval_general():
-        eval_general(
+        return eval_general(
```

Comment:
Why?

Will need to update `eval_io` docstring.

Reply:
To enable comparison of reading results... though I've just thought of a different way to achieve the same thing, might as well try it, too.

Reply:
I've changed the way I test for reading, so this change is no longer necessary; I've reverted that